### DISCUSSION TITLE
Bootstrap 5 Improvement: Forms now require mb-3 on every group?

### DISCUSSION THREAD

**Discussion by SoftCircuits** [twbs/bootstrap#36389] [General] [answered]
Can anyone help me understand why having to add the `mb-3` class to every group in a form is an improvement? In Bootstrap 4, I didn't need to do that and the spacing looked just fine. Are we moving away from convenience?

**Comment by mdo** [comment, marked as answer]
Seems the same as #36385, so marking as answered since I answered there :).
